### ISSUE TITLE
WaterMove subcheck false positive

### ISSUE THREAD

**Issue by Lysandr0** [CaptainObvious0/NoCheatPlus#23]
### Full output of /ncp version
```
[INFO] ---- Version information ----
[INFO] #### Server ####
[INFO] git-Spigot-e6eb36f-d3ed151 ~MC: 1.13.2~
[INFO] detected: 1.13.2
[INFO] #### NoCheatPlus ####
[INFO] Plugin: 3.16.1-SNAPSHOT-CustomFork-v1.3-beta.3
[INFO] MCAccess: 1.4.5-1.12.2|? / CB-Reflect
[INFO] Features:
[INFO] blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10 | BlocksMC1_11 | BlocksMC1_12 | BlocksMC1_13
[INFO] checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency
[INFO] defaults: pvpKnockBackVelocity
[INFO] packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | DebugAdapter
[INFO] Hooks: AllViolations~NCP~ 1.0
[INFO] #### Related Plugins ####
[INFO] ProtocolLib v4.4.0 | ProtocolSupport v4.29-dev
```
### Describe the issue
Swimming right above the surface, will trigger the new sub check

### How to reproduce the issue
Just swim on the surface and you should be able to reproduce it quite easily.

### Extra links/Videos (Including debug logs)
https://www.dropbox.com/s/mhz8i2lagmji5mg/WaterMove_FP.mov?dl=0

[nocheatplus.log](https://github.com/CaptainObvious0/NoCheatPlus/files/3053248/nocheatplus.log)

**Comment by CaptainObvious0** [comment]
Swimming worked fine on my test server. I’ll have to look further into this when I get home.

**Comment by CaptainObvious0** [comment]
Fixed with 34cd09a
